app/src/main/java/ar/com/wolox/android/example/model/UserComplements.java

```diff
@@ -0,0 +1,84 @@
+package ar.com.wolox.android.example.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * UserComplements
+ */
+public class UserComplements {
```

Comment:
This could extend from Serializable. It could be redundant, even though it's a good practice. We are indicating that this class will be filled with data from back